server/src/main/webapp/docs/css/typography.css

```diff
@@ -0,0 +1,14 @@
+/* Google Font's Droid Sans */
+@font-face {
+  font-family: 'Droid Sans';
+  font-style: normal;
+  font-weight: 400;
+  src: local('Droid Sans'), local('DroidSans'), url('../fonts/DroidSans.ttf') format('truetype');
+}
+/* Google Font's Droid Sans Bold */
+@font-face {
+  font-family: 'Droid Sans';
+  font-style: normal;
+  font-weight: 700;
+  src: local('Droid Sans Bold'), local('DroidSans-Bold'), url('../fonts/DroidSans-Bold.ttf') format('truetype');
```

Comment:
Let's just use `font-family: sans-serif` in the CSS so we don't have to package our own fonts.  This [font-face](http://www.paulirish.com/2009/bulletproof-font-face-implementation-syntax/) definition won't even work for a lot of browsers.